ui/ui-components/components/alerts/HydrationError.tsx

```diff
@@ -23,7 +25,19 @@ export default function HydrationError({
         <Card.Header>There was an error loading this Page</Card.Header>
         <Card.Body>
           <blockquote className="blockquote mb-0">
-            <p>{error.message}</p>
+            {error?.message.match(/sign in to continue/) && isBrowser() ? (
```

Comment:
Why only show this on the client? If we're SSRing it should be fine to render on the server.

Reply:
Looking at this, the `pathname` is the only hangup?
Iirc we have a `useRouter` or something from Next, no?
Unless there's something else I've missed ^^

Reply:
Boom - b5be1b3ddac8e13e6ab449d5963a25e236d90b18